### PR TITLE
Fix Setting Window Designs 

### PR DESCRIPTION
**Pull request by onesounds**
**What's the PR**
<strike>- Seperate style to other file from SettingWindow.Xaml (SettingWindow.xaml was too long)</strike>  Let's apply this task when working with valonia.

- Change the Checkboxes to Toggle button in general / Hotkey / Proxy Tab

**Before**

image

**After**

image

**Fix Plugins Designs**
 - Fix Clickable Area
image

 - Fix Hover Color

image

 - Adjust Some Colors
 
image

Fix Store with Darkmode
image
image

**Why?**
- https://uxdworld.com/2018/08/13/checkbox-vs-toggle-switch-7-use-cases-of-forms-design/
- It is not recommended to mix check boxes and toggles. Organize in a toggle for a sense of unity.
- It was a todo that I tried to do in the beginning redesign, but I didn't know how to bind at the time.

**Test Case**
- Each Toggle Button must operate correctly.
- The color shall be marked accurately when you mouse over the plugin item in plugin list
- Even in dark mode, the color shall be displayed accurately.
- There shall be no other effect on the virtual panel.

### CODE REVIEW

File: Flow.Launcher/Flow.Launcher.csproj [hunk 1]
```diff
@@ -116,14 +116,4 @@
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
     <Exec Command="taskkill /f /fi &quot;IMAGENAME eq Flow.Launcher.exe&quot;" />
   </Target>
-
-  <Target Name="RemoveDuplicateAnalyzers" BeforeTargets="CoreCompile">
-    <!-- Work around https://github.com/dotnet/wpf/issues/6792 -->
```
Comment (jjw24) [inline Flow.Launcher/Flow.Launcher.csproj:121]: This work around is no longer required?

File: Flow.Launcher.Test/Flow.Launcher.Test.csproj [hunk 2]
```diff
@@ -10,6 +10,8 @@
     <ApplicationIcon />
     <StartupObject />
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
```
Comment (jjw24) [inline Flow.Launcher.Test/Flow.Launcher.Test.csproj:14]: These two properties are not necessary now that the ci fix is merged in.

File: Flow.Launcher/Flow.Launcher.csproj (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher/Flow.Launcher.csproj:121]: I think so. It should have been fixed in the latest version.

File: Flow.Launcher.Test/Flow.Launcher.Test.csproj (hunk 2 above)
Comment (taooceros) [reply Flow.Launcher.Test/Flow.Launcher.Test.csproj]: Yeah i will remove them.

**Review by taooceros — APPROVED** [review]

**Pull request merged**